src/Mutator/Operator/Coalesce.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Operator;
+
+use Infection\Mutator\Util\Mutator;
+use PhpParser\Node;
+
+/**
+ * @internal
+ */
+final class Coalesce extends Mutator
+{
+    /**
+     * Replaces "'someValue' ?? 'otherValue';" with "(unset) 'someValue' ?? 'otherValue'"
+     *
+     * @param Node $node
+     *
+     * @return Node\Expr\BinaryOp\Coalesce
+     */
+    public function mutate(Node $node)
+    {
+        return new Node\Expr\BinaryOp\Coalesce(new Node\Expr\Cast\Unset_($node->left), $node->right);
```

Comment:
it ought to be
`new Node\Expr\ConstFetch(new Node\Name('null'))`
instead 
`new Node\Expr\Cast\Unset_($node->left)`

Another thing to handle tommorow.

Reply:
Why not simply `$node->right`?

Reply:
good point.